applications/consdbtap/README.md

```diff
@@ -0,0 +1,25 @@
+# consdbtap
+
+IVOA TAP service for the Consolidated Database (ConsDB)
+
+## Source Code
+
+* <https://github.com/lsst-sqre/tap-postgres>
+* <https://github.com/opencadc/tap>
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| cadc-tap.config.backend | string | `"pg"` | What type of backend? |
+| cadc-tap.config.pg.database | string | `"exposurelog"` | Postgres database to connect to |
+| cadc-tap.config.pg.host | string | `"usdf-summitdb.slac.stanford.edu:5432"` | Postgres hostname:port to connect to |
```

Comment:
I think this is OK for now, but very, very soon we want to switch this to usdf-summitdb-replica.

Reply:
I can change the host once it is switched.